packages/components/src/utils/hooks/stories/use-cx.js

```diff
@@ -93,6 +94,72 @@ export const _slotFillSimple = () => {
 	);
 };
 
+// export const _iframe = () => {
+// 	const iFrameRef = useRef( null );
+
+// 	const redText = css`
+// 		color: red;
+// 	`;
+// 	const blueText = css`
+// 		color: blue;
+// 	`;
+
+// 	return (
+// 		<Iframe ref={ iFrameRef }>
+// 			<Example args={ [ redText ] }>This text should be red</Example>
+// 			<StyleProvider document={ iFrameRef.current?.contentDocument }>
+// 				<Example args={ [ blueText ] }>
+// 					This text should be blue
+// 				</Example>
+// 			</StyleProvider>
+// 		</Iframe>
+// 	);
+// };
```

Comment:
Solution attempt no. 1 (manually getting the `Iframe`'s `contentDocument` and passing it to `StyleProvider`)